### PR TITLE
[16.0][IMP] server_action_mass_edit: Improve error message when update record in batch

### PR DESCRIPTION
**Pull request by carlosdauden**
FW-port from https://github.com/OCA/server-ux/pull/631

@Tecnativa
ping @sergio-teruel @chienandalu

### CODE REVIEW

**Review by chienandalu — COMMENTED** [review]
Looks like something else might be missing as the installation fails in the CI

**Review by sergio-teruel — APPROVED** [review]
Functional review ok

File: server_action_mass_edit/wizard/mass_editing_wizard.py [hunk 1]
```diff
@@ -281,9 +299,44 @@ def create(self, vals_list):
                             values.update({split_key: m2m_list})
 
                 if values:
```
Comment (chienandalu) [inline server_action_mass_edit/wizard/mass_editing_wizard.py]: Maybe you can avoid the complexity warning `C901` like this
```suggestion
                if not values:
                    continue
```

Comment (carlosdauden) [reply server_action_mass_edit/wizard/mass_editing_wizard.py]: Changed, but `noqa: C901` already is needed

Comment (carlosdauden) [reply server_action_mass_edit/wizard/mass_editing_wizard.py]: @chienandalu is OK?

**Review by chienandalu — APPROVED** [review]

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**